kratos/tests/test_variable_utils.py

```diff
@@ -1,4 +1,5 @@
 ﻿# Importing the Kratos Library
+from unittest.case import expectedFailure

```

Comment:
where is this used?

Reply:
No idea. Actually I don't know why it appears as a change.

Reply:
Removed. Most probably I added it by accident.